components/sync15/android/src/main/java/mozilla/appservices/sync15/SyncTelemetryPing.kt

```diff
@@ -33,11 +34,13 @@ enum class FailureName {
 
 data class SyncTelemetryPing(
     val version: Int,
-    val uid: String?,
+    val uid: String,
     val events: List<EventInfo>,
     val syncs: List<SyncInfo>
 ) {
     companion object {
+        @JvmField val EMPTY_UID = "0".repeat(32)
```

Comment:
I guess this is how Kotlin does static fields with non-constant initializers? I cribbed that from [this post](https://blog.egorand.me/where-do-i-put-my-constants-in-kotlin/).